Sources/Mantis/CropView/CropView.swift

```diff
@@ -700,6 +717,20 @@ extension CropView {
     func crop() -> CropOutput {
         return crop(image)
     }
+    
+    /// completion is called in the main thread
+    func asyncCrop(completion: @escaping (_ cropOutput: CropOutput) -> Void ) {
+        activityIndicator.isHidden = false
+        activityIndicator.startAnimating()
+        
+        DispatchQueue.global(qos: .userInteractive).async {
```

Comment:
Do we also need?

```suggestion
        DispatchQueue.global(qos: .userInteractive).async { [weak self] in
```

Reply:
No it is not needed. DispatchQueue does not hold strong self to it won't cause retain cycle.
You can test it by putting a breakpoint in `deinit`